runtime/onert/core/src/compiler/pass/ConstantInsertionPass.cc

```diff
@@ -40,9 +40,25 @@ void ConstantInsertionPass::callback(const ir::OperationIndex &node_index, ir::I
     auto &object = _graph.operands().at(input);
 
     const auto key = ReplaceKey{input, factor};
-    if (object.isConstant() && (object.getUses().size() >= 2 ||
-                                _replace_operands_map.find(key) != _replace_operands_map.end()))
+
+    // Skip if the operand is not constant or has only one use
+    if (!object.isConstant() || object.getUses().size() < 2)
+      continue;
```

Comment:
@ragmani 

>As far as I know, ConstantInsertionPass has had one role that inserts operands only when circle model has shared constant.
From now on, are you going to allow ConstInsertionPass to insert all required constant operands as well.

Constant insertion is skipped if constant is not shared (`object.getUses().size() < 2`)

Reply:
> are you going to allow ConstInsertionPass to insert all required constant operands as well.

I didn't think about usage of this pass. I updated this to handle all operand possible case for easy understood of this pass's objective.

Reply:
> I updated this to handle all operand possible case for easy understood of this pass's objective.

It looks good to me.